.github/workflows/codespell.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Codespell
+        uses: codespell-project/actions-codespell@v1
```

Comment:
Actually I think we would like to exclude the jupyter notebooks, since it seems to pick up a number of things that it shouldn't change. Would that be possible?